model/src/main/java/org/jboss/pnc/model/RepositoryConfiguration.java

```diff
@@ -61,6 +79,15 @@
     private String internalUrl;
 
     /**
+     * Normalized version of scm url to query against. Normalized version is without the protocol and .git extension.
+     */
+    @Size(max = 255)
+    @Column(unique = true, updatable = false)
+    @Getter
+    @Setter
+    private String internalUrlNormalized;
```

Comment:
Why do you want to create another field just to strip out the suffix?
I think it's unnecessary data duplication.

I'd just update the URL to strip it out and store it normalized